src/commands/init.rs

```diff
@@ -77,10 +83,88 @@ complete -F _{function_name}_completions {function_name}
     };
 }
 
-// given the location of the cookbook executable, return
-// back the init script for cookbook.
 
-pub fn init(cookbook_executable: &str, mut args: Peekable<Iter<String>>) -> Result<String, String> {
+macro_rules! fish_init_body {
+    () => {
+        r#"
+function __fish_tome_help_message
```

Comment:
Fish shell is really nice and simple for completions. They're standard functions and they are pretty easy to debug. Also the UI of it is pretty ⭐